{"failedTests":[{"testName":"sequential/test-worker-prof","timestamp":1759707572822,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win11-arm64-COMPILED_BY-vs2022_clang-arm64 #37130","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37130/","buildNumber":37130,"builtOn":"test-azure_msft-win11_vs2022-arm64-2","nodeVersion":"25.0.0","commitHash":"875aff9384bd0062cda388dc09b0b520123064c7","callStack":[{"upstreamBuild":73430,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":83164,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":69574,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"60061"}],"tap":{"ok":false,"name":"sequential/test-worker-prof","id":1142,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"files found in working directory before worker starts [ 'isolate-000001F2E8014000-13804-v8.log' ]\nparent prof file: isolate-000001F2E8014000-13804-v8.log\nparent posted 37731 messages\nfiles found in working directory before worker exits [\n  'isolate-000001F2E8014000-13804-v8.log',\n  'isolate-000001F2F0D98000-13804-v8.log'\n]\nworker prof file: isolate-000001F2F0D98000-13804-v8.log\n\nworker ticks 13\nparent ticks 50\nnode:internal/assert/utils:77\n    throw err;\n    ^\n\nAssertionError [ERR_ASSERTION]: worker ticks <= 15:\ntick,0x7ff7bb7358e4,43014,0,0x0,2\ntick,0x7ff7bb7e5b2c,90993,0,0x0,0,0x2163ee96dd8,0x1f290082e90,0x1f2900829d4,0x2163ee969f0,0x2163ee8c516\ntick,0x7ff85ed6cb0c,378984,0,0x0,0,0x1f2900c0318,0x1f290048218,0x1f2900c0528\ntick,0x7ff85ed30e74,522980,0,0x0,0,0x1f2900c0318,0x1f290048218,0x1f2900c0528\ntick,0x7ff85ed30e74,554983,0,0x0,0,0x1f2900c0318,0x1f290048218,0x1f2900c0528\ntick,0x7ff85ed30e74,746980,0,0x0,0,0x1f2900c0318,0x1f29008c6ec,0x1f2900c0528\ntick,0x7ff85ed30e74,938983,0,0x0,0,0x1f29008d04c,0x1f29008c6ec,0x1f2900c0528\ntick,0x7ff7bbc000a8,971006,0,0x0,0,0x1f29008d04c,0x1f29008c6ec,0x1f2900c0528\ntick,0x7ff7bba48d74,1034991,0,0x0,5\ntick,0x7ff7bba48fe4,1274980,0,0x0,5\ntick,0x7ff7bbc56338,1354979,0,0x0,0,0x1f29008d04c,0x1f29008c6ec,0x1f2900c0528\ntick,0x7ff85ed30e74,1578986,0,0x0,0,0x1f29008d04c,0x1f29008c6ec,0x1f29008d228\ntick,0x7ff7bbad0c8c,1594991,0,0x0,0\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\sequential\\test-worker-prof.js:81:3)\n    at Module._compile (node:internal/modules/cjs/loader:1759:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1892:10)\n    at Module.load (node:internal/modules/cjs/loader:1479:32)\n    at Module._load (node:internal/modules/cjs/loader:1298:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:328:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)\n    at node:internal/main/run_main_module:33:47 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: false,\n  expected: true,\n  operator: '==',\n  diff: 'simple'\n}\n\nNode.js v25.0.0-pre"},"time":2718.006,"fullname":"sequential/test-worker-prof","closingTestPoint":false}},{"testName":"parallel/test-runner-run-watch","timestamp":1759707572822,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022_clang #37130","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022_clang/37130/","buildNumber":37130,"builtOn":"test-rackspace-win2022_vs2022-x64-3","nodeVersion":"25.0.0","commitHash":"875aff9384bd0062cda388dc09b0b520123064c7","callStack":[{"upstreamBuild":73430,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":83164,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":69574,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"60061"}],"tap":{"ok":false,"name":"parallel/test-runner-run-watch","id":900,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"Test failure: 'should run tests repeatedly'\nLocation: test\\parallel\\test-runner-run-watch.mjs:198:3\nAssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n\n'# Subtest: test.js\\n' +\n  'not ok 2 - test.js\\n' +\n  '  ---\\n' +\n  '  duration_ms: 1058.3104\\n' +\n  \"  type: 'test'\\n\" +\n  \"  location: 'C:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.787\\\\\\\\test.js:1:1'\\n\" +\n  \"  failureType: 'testCodeFailure'\\n\" +\n  '  exitCode: ~\\n' +\n  \"  signal: 'SIGTERM'\\n\" +\n  \"  error: 'test failed'\\n\" +\n  \"  code: 'ERR_TEST_FAILURE'\\n\" +\n  '  ...\\n' +\n  '# Subtest: test has ran\\n' +\n  'ok 2 - test has ran\\n' +\n  '  ---\\n' +\n  '  duration_ms: 1.8433\\n' +\n  \"  type: 'test'\\n\" +\n  '  ...\\n' +\n  '1..2\\n' +\n  '# tests 2\\n' +\n  '# suites 0\\n' +\n  '# pass 1\\n' +\n  '# fail 1\\n' +\n  '# cancelled 0\\n' +\n  '# skipped 0\\n' +\n  '# todo 0\\n' +\n  '# duration_ms 440.7945\\n'\n\n    at testUpdate (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:91:14)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async testWatch (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:187:26)\n    at async TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:199:5)\n    at async Test.run (node:internal/test_runner/test:1113:7)\n    at async Promise.all (index 0)\n    at async Suite.run (node:internal/test_runner/test:1516:7)\n    at async startSubtestAfterBootstrap (node:internal/test_runner/harness:358:3) {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: '# Subtest: test.js\\n' +\n    'not ok 2 - test.js\\n' +\n    '  ---\\n' +\n    '  duration_ms: 1058.3104\\n' +\n    \"  type: 'test'\\n\" +\n    \"  location: 'C:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.787\\\\\\\\test.js:1:1'\\n\" +\n    \"  failureType: 'testCodeFailure'\\n\" +\n    '  exitCode: ~\\n' +\n    \"  signal: 'SIGTERM'\\n\" +\n    \"  error: 'test failed'\\n\" +\n    '...',\n  expected: /tests 1/,\n  operator: 'match',\n  diff: 'simple'\n}"},"time":25235.84,"fullname":"parallel/test-runner-run-watch","closingTestPoint":false}}]}